# KirimKecepatanUdp.msg
# Message untuk mengirim data kecepatan via UDP

float32 kecepatan_x
float32 kecepatan_y
float32 kecepatan_sudut
float32 sudut_servo
